src/find.ts

```diff
@@ -1,7 +1,7 @@
 import { join } from 'node:path';
 import { existsSync } from 'node:fs';
 
-import * as walk from 'empathic/walk';
+import { up as walkUp } from 'empathic/walk';
```

Comment:
sigh

Reply:
Again, better for tree shaking

Reply:
No, it's not.

Reply:
Its only true when the target module has top level side effects. Walk is pure functions and EVEN if/in worst case, walk only has `walk.up` which is all/what we want anyway.

Bundlers are much smarter now than they used to be. This is 10000% fine.

Reply:
Simply easier to tree shake than what you did